src/main/webapp/app/exam/manage/exam-management.route.ts

```diff
@@ -814,14 +805,23 @@ export const examManagementRoute: Routes = [
         canActivate: [UserRouteAccessService],
     },
     {
-        path: ':examId/exercise-groups/:exerciseGroupId/modeling-exercises/:exerciseId/submissions/:submissionId/assessments/:resultId',
```

Comment:
The navigation.utils.ts add the `s/resultId` in lines 132-135. Did you check if this change has an influence on the navigation there?

Reply:
Yes. This was actually duplicated in the routing and still exists in line 853.

Reply:
I looked at the course routes where the same authorities get used, so this should be fine. I also think it makes more sense that way, considering that tutors should not assess a specific result (double blind).
![grafik](https://github.com/ls1intum/Artemis/assets/26540346/d71f3292-b5f0-48f9-815e-88503b82d7b2)